### PR TITLE
Feature/86b09hx47 api for schedule

### PR DESCRIPTION
**Pull request by knordstrom**
This is the core of the demo, an API that produces a description of the user's schedule over a specified time period

### CODE REVIEW

File: docker/docker-compose.yml
Comment (mshash8) [inline docker/docker-compose.yml:1]: @knordstrom I needed to add the line:
```- NEO4J_AUTH=neo4j/password``` under "environment" for authorization. Did you find any other way to do the auth?

File: library/neo4j.py
Comment (mshash8) [inline library/neo4j.py:1]: @knordstrom I think I see the auth here. It would be line 14 right?

File: docker/docker-compose.yml
Comment (knordstrom) [reply docker/docker-compose.yml:1]: I don't remember having to do too much but this is the right way to do it. Environment variables (or later in prod a vaulting solution) will be how we do this.

Good find.

File: library/neo4j.py
Comment (knordstrom) [reply library/neo4j.py:1]: Yeah, that's right. I realized after I commented that I just actually set passwords in my neo4j console http://localhost:7474/browser/ but I that's not really the best way to do things. I wasn't really paying attention, your changes in that regard will be better.

**Pull request merged**